JSONModel/JSONModel/JSONModel.m

```diff
@@ -721,6 +718,29 @@ -(void)__inspectProperties
                              );
 }
 
+-(void)__addCustomSetterForName:(NSString*) name
+                     withSuffix:(NSString*) suffix
+                            key:(NSString*) key
+                     toProperty:(JSONModelClassProperty*) p{
```

Comment:
Please change to something more like:

```objc
- (void)__addCustomSetterToProperty:(JSONModelClassProperty *)property withValueType:(NSString *)valueType
{
```

- all args/params on a single line
- curly brace on a new line
- space before pointer asterisks, rather than after
- no space before variable names
- more clear variable/param naming
- avoiding unneeded params